scripts/genappimage.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+########################################################################
+# Package the baries built on Travis-CI as an AppImage
+# By Simon Peter 2016
+# For more information, see http://appimage.org/
+########################################################################
+
+if [ -z "$ARCH" ]; then
+  export ARCH="$(arch)"
+fi
+
+APP=NVim
+LOWERAPP="${APP,,}"
```

Comment:
It also looks like the only place you need bash, except that I do not know what appimage_functions.sh supports.